File: Revision_classes/06.Dv, LVP,PassByValue/Test5.java
class Test5 {
	static int a = 10;

	public static void main(String[] args) {
		
		System.out.println(a);
		System.out.println(Test5.a);
		System.out.println();

		int  a = 50;
		System.out.println(a);
		System.out.println(Test5.a);
		System.out.println();

		a = 60;
		System.out.println(a);
		System.out.println(Test5.a);
		System.out.println();

		Test5.a = 70;
		System.out.println(a);
		System.out.println(Test5.a);
		System.out.println();
		
/*
		b = 90;
		System.out.println(b);
		System.out.println(Test3.b);
		*/

	}
}

File: React/app7/src/Test5.js
export default function Test5() {
    const w1 = (resolve, reject) => {
        //perform the logic and assume results as 1000
        //return resolve(1000);        
        return reject('something went wrong');
    }
    const test = () => {
        const p1 = new Promise(w1);
        p1.then((res) => console.log(res))
        .catch((err) => console.log(err));
    }
    return (
        <>
            <button type="button" onClick={test}>test</button>
        </>
    );
}